lms/djangoapps/commerce/urls.py

```diff
@@ -4,10 +4,10 @@
 
 from django.conf.urls import patterns, url
 
-from .views import OrdersView, checkout_cancel
+from .views import BasketsView, checkout_cancel
 
 urlpatterns = patterns(
     '',
-    url(r'^orders/$', OrdersView.as_view(), name="orders"),
+    url(r'^baskets/$', BasketsView.as_view(), name="baskets"),
```

Comment:
Actually, looking more closely at that code, I'm guessing you'll need to update this as well:
https://github.com/edx/edx-platform/blob/master/lms/static/js/student_account/enrollment.js#L12